frontend/pages/admin/UserManagementPage/components/UserForm/UserForm.tsx

```diff
@@ -423,189 +472,208 @@ const UserForm = ({
           </>
         }
       />
-      {!isNewUser &&
-        !isInvitePending &&
-        isModifiedByGlobalAdmin &&
-        !formData.sso_enabled && (
-          <div className={`${baseClass}__edit-password`}>
-            <div className={`${baseClass}__password`}>
-              <InputField
-                label="Password"
-                error={errors.password}
-                name="password"
-                onChange={onInputChange("password")}
-                placeholder="••••••••"
-                value={formData.password || ""}
-                type="password"
-                helpText="12-48 characters, with at least 1 number (e.g. 0 - 9) and 1 symbol (e.g. &*#)."
-                blockAutoComplete
-              />
-            </div>
-          </div>
-        )}
-      {((!isNewUser && formData.sso_enabled) || canUseSso) && (
-        <div className={`${baseClass}__sso-input`}>
-          <Checkbox
-            name="sso_enabled"
-            onChange={onCheckboxChange("sso_enabled")}
-            value={formData.sso_enabled}
-            readOnly={!canUseSso}
-            wrapperClassName={`${baseClass}__invite-admin`}
-            helpText={
-              canUseSso ? (
-                <p className={`${baseClass}__sso-input sublabel`}>
-                  Password authentication will be disabled for this user.
-                </p>
-              ) : (
-                <span className={`${baseClass}__sso-input sublabel-nosso`}>
-                  This user previously signed in via SSO, which has been
-                  globally disabled.{" "}
-                  <button className="button--text-link" onClick={onSsoDisable}>
-                    Add password instead
-                    <Icon
-                      name="chevron-right"
-                      color="core-fleet-blue"
-                      size="small"
-                    />
-                  </button>
-                </span>
-              )
-            }
-          >
-            Enable single sign-on
-          </Checkbox>
-        </div>
+    </>
+  );
+
+  const renderAuthenticationSection = () => (
+    <div className="form-field">
+      <div className="form-field__label">Authentication</div>
+      <Radio
+        className={`${baseClass}__radio-input`}
+        label="Enable single sign-on"
+        id="enable-single-sign-on"
+        checked={formData.sso_enabled}
+        value={formData.sso_enabled.toString()}
```

Comment:
would `!!formData.sso_enabled` be a better logical match to the `value` field of the companion radio?